.github/workflows/ci.yaml

```diff
@@ -85,32 +85,15 @@ jobs:
       - name: Cache Rust artifacts
         uses: actions/cache@v2
         if: |
-          matrix.cmd == 'run-cargo-clippy' || matrix.cmd == 'run-functions-examples --application-variant=rust'
+          matrix.cmd == 'run-cargo-clippy' || matrix.cmd == 'run-examples --application-variant=rust' || matrix.cmd == 'run-examples --application-variant=cpp' || matrix.cmd == 'run-functions-examples --application-variant=rust'
```

Comment:
nit: Could this be split into multiple shorter lines?

Reply:
I tried before and got it wrong and it would silently ignore parts of the string, so I'll keep it like this for now, I will remove it at some point though when the cache can contain everything